src/orthopolybasis.jl

```diff
@@ -219,26 +220,26 @@ end
 
 
 # ------------------   rrules 
-
 # 
 # ∂_xa ( ∂P : P ) = ∑_ij ∂_xa ( ∂P_ij * P_ij ) 
 #                 = ∑_ij ∂P_ij * ∂_xa ( P_ij )
 #                 = ∑_ij ∂P_ij * dP_ij δ_ia
 #
-function rrule_evaluate!(P::AbstractArray, basis::OrthPolyBasis1D3T, X::AbstractVector)
-   nX = length(X) 
-   dP = similar(P)
-   evaluate_ed!(P, dP, basis, X)
-
-   function pb(∂P)
-      ∂X = zeros(nX)
-      for j = 1:length(basis) 
-         for i = 1:nX 
-            ∂X[i] += ∂P[i, j] * dP[i, j]
-         end
-      end
-      return ∂X 
-   end
-
-   return P, pb 
-end
+# function ChainRulesCore.rrule(::typeof(evaluate), basis::OrthPolyBasis1D3T, x::AbstractVector)
```

Comment:
should these comments be removed?

Reply:
I didn't delete that since that was not written by me. I will remove that in the next update.